extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -16,12 +16,14 @@ import * as episodeIcons from '../icons/episode-icons';
 import { STATE_ERROR, STATE_PLAYING } from '../constants';
 
 const TrackIcon = ( { isPlaying, isError, className } ) => {
-	let name;
+	let hiddenText, name;
 
 	if ( isError ) {
 		name = 'error';
+		hiddenText = __( 'Error:' );
 	} else if ( isPlaying ) {
 		name = 'playing';
+		hiddenText = __( 'Playing: ' );
```

Comment:
```suggestion
		hiddenText = __( 'Playing: ', 'jetpack' );
```

Reply:
Also wondering whether you should be translating the `:` or leaving that out and concatenating it on later.

Reply:
No idea on that one. I think it does give some additional context on how it will be handled though by leaving the `:` in there. That said, I fully defer to people who actually know what they're talking about regarding translatable strings 😅

Reply:
I think the formatting should be included in the translated string too.